tooling/CurrentVersion.props

```diff
@@ -1,6 +1,6 @@
 <!-- [auto-generated] This file is automatically re-created when Bonsai releases and generally should not be modified by hand [/auto-generated] -->
 <Project>
   <PropertyGroup>
-    <BonsaiVersion>2.8.6</BonsaiVersion>
```

Comment:
Not a huge deal, but we don't really need to manually bump this. It's fine if seeing the "wrong" version was making you itchy though.

(Only real issue is it could cause minor confusion if we had an unplanned 2.8.6 like we did with 2.8.5.)

Reply:
Yup, in general I am fine with just following the auto-tagging logic. The only reason I did this here is that this time we want to start previewing the new editor features as soon as they are merged into main this time, since it's hard to make a good estimate of when 2.9 will come out, and I wanted people to already be seeing the 2.9 version in there to avoid confusion.

I thought that the .NET core PR would be a meaningful place to do the bump since I would like to merge it before the other big feature PRs (so I can test them also against .NET core) and I think this PR will always be a one-way street anyway. With all the changes we did to target frameworks, updated dependencies and functionality changes I don't think we could ever call a version 2.8.x once this is merged into main.

Reply:
Makes sense to me!